modules/queue-manager/variables.tf

```diff
@@ -19,9 +19,9 @@ variable "name" {
 
 # version is reserved attribute on modules
 variable "queue_manager_version" {
-  description = "The MQ version of the queue manager."
+  description = "The MQ version of the queue manager. If null, use the latest available"
```

Comment:
I suggest to include details on how user can find a list of versions (maybe copy what the provider says)

Reply:
Also "If null" will be confusing for project consumers. Perhaps reword it to something like, "If no value set..."